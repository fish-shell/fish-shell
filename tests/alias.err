
####################
# Avoid regressions of issue #3860 wherein the first word of the alias ends with a semicolon

####################
# Bare `alias` should list the aliases we have created and nothing else
