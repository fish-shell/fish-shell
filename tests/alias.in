logmsg Avoid regressions of issue \#3860 wherein the first word of the alias ends with a semicolon
function foo
    echo ran foo
end
alias my_alias "foo; and echo foo ran"
my_alias

alias a-2='echo "hello there"'

logmsg Bare `alias` should list the aliases we have created and nothing else
# We have to exclude two aliases because they're an artifact of the unit test
# framework and we can't predict the definition.
alias | grep -Ev '^alias (fish_indent|fish_key_reader) '
