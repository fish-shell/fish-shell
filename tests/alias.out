
####################
# Avoid regressions of issue #3860 wherein the first word of the alias ends with a semicolon
ran foo
foo ran

####################
# Bare `alias` should list the aliases we have created and nothing else
alias a-2 'echo "hello there"'
alias my_alias 'foo; and echo foo ran'
